packages/renderer/src/lib/images/LoaderAnimation.svelte

```diff
@@ -0,0 +1,136 @@
+<script lang="ts">
+import { onMount } from 'svelte';
+
+export let size = 400;
+
+let dotsGroup: SVGGElement | null = null;
+let eyesElement: SVGUseElement | null = null;
+
+onMount(() => {
+  // Configuration - scale from 800pt design
+  const scale = size / 800;
+  const numDots = 110;
+  const radius = 394 * scale;
+  const dotRadius = 5 * scale;
+  const dotColor = '#cb5839';
+
+  if (dotsGroup) {
+    for (let i = 0; i < numDots; i++) {
+      const angle = (i * 2 * Math.PI) / numDots;
+      const circle = document.createElementNS('http://www.w3.org/2000/svg', 'circle');
+
+      circle.setAttribute('cx', (radius * Math.cos(angle)).toFixed(2));
+      circle.setAttribute('cy', (radius * Math.sin(angle)).toFixed(2));
+      circle.setAttribute('r', dotRadius.toString());
+      circle.setAttribute('fill', dotColor);
+
+      // eslint-disable-next-line svelte/no-dom-manipulating
+      dotsGroup.appendChild(circle);
+    }
+  }
+
+  function blink(): void {
+    if (eyesElement) {
+      eyesElement.style.transition = 'opacity 0.05s ease-in';
+      eyesElement.style.opacity = '1';
+
+      setTimeout(() => {
+        if (eyesElement) {
+          eyesElement.style.transition = 'opacity 0.15s ease-out';
+          eyesElement.style.opacity = '0';
+        }
+      }, 80);
+
+      // eslint-disable-next-line sonarjs/pseudo-random
+      setTimeout(blink, 500 + Math.random() * 2000);
```

Comment:
suggestion: numbers > 1000 are written using the format 1_000, 2_000 in the codebase